docs/_articles/en/codepreview/set-up-lwc-code-preview.md

```diff
@@ -12,6 +12,19 @@ Run this command from a command-line interface.
 ```
 sfdx plugins:install @salesforce/lwc-dev-server
 ```
+
 > **Troubleshooting Tip:** Due to a known issue, it's likely you'll see several errors when you install the plug-in. Run `sfdx plugins --core` to see if the plug-in is installed. If yes, try to start the Code Preview server. If successful, you're good to proceed.
 
 After you select **SFDX: Preview Component**, the Command Palette displays a list of preview options. You can choose to preview your component in the desktop browser or in a virtual mobile device (iOS or Android). Mobile previews require additional setup. See [Preview Lightning Web Components on Mobile](https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.mobile_extensions) in the _Lightning Web Components Dev Guide_.
+
+## Recommended Settings
+
+**Disable Browser Cache**  
+If you would like to see changes made to your static resources in real time, please disable the web browser cache when working with LWC Code Preview.
+
+For some examples of how to disable the web browser cache, please see...
```

Comment:
FYI that we only use "please" if we're asking the customer to do something really unpleasant, like reinstall all the tooling or something similarly yucky.